awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -269,9 +282,18 @@ class JobOutput extends Component {
     this.scrollToRow(0);
   }
 
-  handleScrollLast() {
-    const { remoteRowCount } = this.state;
-    this.scrollToRow(remoteRowCount - 1);
+  async handleScrollLast() {
+    const { job, type } = this.props;
+    const {
+      data: {
+        results: [{ counter }],
+      },
+    } = await JobsAPI.readEvents(job.id, type, {
+      page_size: 1,
+      order_by: '-counter',
+    });
```

Comment:
Given a job with 40012 events on our internal test instance, this seems to add about ~2s to the total loading time when clicking `page last` (on average).

![Screenshot from 2020-01-16 12-04-04](https://user-images.githubusercontent.com/9753817/72546486-d6c91000-3858-11ea-9719-cc48d0fa327f.png)

Reply:
This is slow because `counter` isn't indexed.